### PR TITLE
Pull 2015 schedule from OSEM dynamically

### PR DESCRIPTION
**Pull request by kormoc**
This change pulls in the OSEM schedule dynamically via jsonp to live update on the website.

### CODE REVIEW

File: schedule/2015.html [hunk 1]
```diff
@@ -8,8 +8,188 @@
...
-</div><!--/span-->
+<link rel="stylesheet" type="text/css" media="screen" href="/js/featherlight-1.3.4/featherlight.min.css">
+<script src="/js/featherlight-1.3.4/featherlight.min.js"></script>
+
+
+<script>
+function create_schedule(result) {
+    var schedule = result;
+    var times = {};
+    var rooms = new Array();
```
Comment (meonkeys) [inline schedule/2015.html]: I think `var rooms = [];` is canon, but I'm fairly new to JavaScript.

File: schedule/2015.html [hunk 2]
```diff
@@ -8,8 +8,188 @@
...
+        });
+    });
+}
+
+$(function(){
+    $.ajax({
+        url:        "https://osem.seagl.org/conference/seagl2015/schedule.json",
+        jsonp:      "callback",
+        dataType:   "jsonp",
+        success:    create_schedule
```
Comment (meonkeys) [inline schedule/2015.html]: Probably good form to add an `error` callback that pops up to tell the user there's some problem contacting osem, or at least logs same using `console.error()`.

File: schedule/2015.html [hunk 3]
```diff
@@ -8,8 +8,188 @@
...
+            .addClass('col-sm-'+col_size)
+            .addClass('col-md-'+col_size)
+            .addClass('col-lg-'+col_size)
+            .append($('<h3/>').text(this.name))
+        );
+
+        // Build the list of the talks per timeslot
+        $(this.events).each(function() {
+            if ($.inArray(this.start_time, Object.keys(times)) == -1) {
+                times[this.start_time] = new Array();
```
Comment (meonkeys) [inline schedule/2015.html]: again, `times [this.start_time] = [];` ... I just feel like I've seen the short form more often. Again, could be wrong. :)

File: .gitignore [hunk 4]
```diff
@@ -5,3 +5,4 @@ _site
 atlassian-ide-plugin.xml
 *.iml
 *.idea
+Gemfile.lock
```
Comment (meonkeys) [inline .gitignore]: Why? What's this?

**Comment by meonkeys** [comment]
Looks good. Left a few comments, then I'm thinking :shipit:

File: schedule/2015.html (hunk 2 above)
Comment (kormoc) [inline schedule/2015.html]: Fixed in d11a17d4536063dbe49d6332b4ab4b9cffda30fa

File: schedule/2015.html (hunk 3 above)
Comment (kormoc) [inline schedule/2015.html]: Fixed in 71f9bb932c7d7de60f7ca0eb264b1621c464e68b

File: schedule/2015.html (hunk 1 above)
Comment (kormoc) [inline schedule/2015.html]: Fixed in 71f9bb932c7d7de60f7ca0eb264b1621c464e68b

[] is just an alias for new Array(), however you are correct that the vast majority of folks just use [], as any browser used today does support it.

File: .gitignore (hunk 4 above)
Comment (kormoc) [inline .gitignore]: When you build the github-pages gem, it sets a lockfile with the versions it built. There's no need to sync those versions between us, so it's best to ignore this file.

File: schedule/2015.html [hunk 5]
```diff
@@ -6,10 +6,210 @@
...
+<div class="col-md-12" id='schedule_content'>
     <h1>SeaGL 2015 Schedule</h1>
+    <div class="container-fluid" id="schedule_div">
+        <div class="row" id="rooms_row"></div>
+    </div>
+</div>
 
-    <p>The official SeaGL schedule is published
-    <a href="https://osem.seagl.org/conference/seagl2015/schedule">here</a>.
-    </p>
```
Comment (AndrewKvalheim) [inline schedule/2015.html]: I'd leave this message in as a fallback for readers who don't use JavaScript and for when the schedule fails to load.

File: schedule/2015.html [hunk 6]
```diff
@@ -6,10 +6,210 @@
...
+        var time_row_id = 'row_time_'+timestamp;
+
+        // Add in a HR to make it visually split
+        $('#schedule_div').append($('<hr/>'));
+        // Add in the time row
+        $('#schedule_div').append(
+            $('<div/>')
+            .addClass('row')
+            .attr('id', time_row_id)
+            )
```
Comment (AndrewKvalheim) [inline schedule/2015.html]: missing semicolon

File: schedule/2015.html (hunk 5 above)
Comment (kormoc) [inline schedule/2015.html]: Good idea.

Fixed in 5e2c9f7538a4a3fd281e044815e784212a681db3

I've added <noscript> and also use that content in the error handler so failures are not as critical

**Pull request merged**